packages/rum-core/test/performance-monitoring/transaction-service.spec.js

```diff
@@ -351,14 +357,6 @@ describe('TransactionService', function() {
     })
   })
 
-  it('should ensureCurrentTransaction once per startTransaction', function() {
```

Comment:
Would you please revert this removal? we used to have a bug that we called the created transaction multiple times and that would result in loosing spans.

Reply:
Done